cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,66 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 2.6.3 <http://docs.cask.co/cdap/2.6.3/index.html>`__
+=============================================================
+
+CDAP Bug Fixes
+--------------
+
+- Replace use of CDAP transaction snapshot codecs with the Tephra versions instead
+  (`CDAP-2496 <https://issues.cask.co/browse/CDAP-2496>`__)
+  
+  
+Upgrade Instructions
+--------------------
+
+- **Update Memory Setting for Transaction Service before Upgrading**
+
+  As part of the upgrade, we recommend that the memory configuration for transaction
+  service be increased from its default value of 512MB. We suggest adjusting it in
+  ``cdap-site.xml`` to 4096mb::
+
+    <property>
+    <name>data.tx.memory.mb</name>
+    <value>4096</value>
+    </property>
+
+- **Upgrading from 2.6.2 to 2.6.3**
```

Comment:
This should just be our standard upgrade instructions. No need to have special one. I suggest we remove this altogether